integration_tests/suite/test_cdr.py

```diff
@@ -358,6 +577,214 @@ def test_given_no_call_logs_when_list_cdr_then_empty_list(self):
+            ),
+            Destination(
+                destination_details_key='user_name',
+                destination_details_value='Alice Wonderland',
+            ),
+        ],
+        participants=[
+            {
+                'user_uuid': USER_1_UUID,
+                'line_id': '100',
+                'tags': ['Davy Jones', 'Locker'],
+                'role': 'source',
+                'answered': True,
+            },
+            {
+                'user_uuid': USER_2_UUID,
+                'line_id': '200',
+                'tags': ['Wonderland'],
+                'role': 'destination',
+                'answered': True,
+            },
+        ],
+        recordings=[],
+    )
+    @call_log(
+        **{'id': 2},
+        date='2022-07-24 00:00:00',
+        date_answer='2022-07-24 00:01:00',
+        date_end='2022-07-24 00:02:27',
+        destination_exten='*41610342',
+        destination_name='Meeting with Jack Sparrow',
+        direction='internal',
+        requested_exten='*41610342',
+        requested_context='mycontext1',
+        requested_internal_context='mycontext1',
+        source_internal_name='Jack Sparrow',
+        source_exten='1602',
+        source_name='Jack Sparrow',
+        source_internal_exten='1602',
+        source_internal_context='mycontext1',
+        destination_details=[
+            Destination(
+                destination_details_key='type',
+                destination_details_value='meeting',
+            ),
+            Destination(
+                destination_details_key='meeting_uuid',
+                destination_details_value='6648726e-8ed9-4e6e-8ea5-f63caf911ae9',
+            ),
+            Destination(
+                destination_details_key='meeting_name',
+                destination_details_value='Meeting with Jack Sparrow',
+            ),
+        ],
+        recordings=[],
+    )
+    @call_log(
+        **{'id': 3},
+        date='2022-07-24 00:00:00',
+        date_answer='2022-07-24 00:01:00',
+        date_end='2022-07-24 00:02:27',
+        destination_exten='1901',
+        destination_name='myconference1',
+        direction='internal',
+        requested_exten='1901',
+        requested_context='mycontext2',
+        requested_internal_context='mycontext2',
+        source_internal_name='Jack Sparrow',
+        source_exten='1602',
+        source_name='Jack Sparrow',
+        source_internal_exten='1602',
+        source_internal_context='mycontext2',
+        destination_details=[
+            Destination(
+                destination_details_key='type',
+                destination_details_value='conference',
+            ),
+            Destination(
+                destination_details_key='conference_id',
+                destination_details_value='2',
+            ),
+        ],
+        recordings=[],
+    )
+    def test_given_call_logs_with_destination_details_when_list_cdr_then_list_cdr_has_remote_party(
```

Comment:
The name `remote_party` does not exist anymore.

Reply:
yes, you're right, I missed that.